addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -63,6 +63,11 @@ def get_mail_values(self, res_ids):
                                     height: unset !important;
                                 }"""
                         )
+                        complete_body_html = complete_body_html.replace(
+                            'td > img:only-child {',
+                            """img:only-child:not(.img-fluid) {
+                                    object-fit: cover;"""
+                        )
```

Comment:
This construction is used three times. If we ever need to make a fix in it there is a high probability we'll miss the other two, especially if it's someone else. Isn't there a way we could have it at one place and be done with it ? Why do we have to do it multiple times ?

Reply:
I don't remember exactly, is this change in its own separate commit ? If not then it should be so that it can be adapted properly to be clean for master.

Reply:
The three versions of this construction are all in this commit if that's what you mean.

Reply:
But is it a single commit with nothing else or few other things  ?

Reply:
A few other things that require this change to work.